runner/arazzo_runner/extractor/openapi_extractor.py

```diff
@@ -513,9 +520,16 @@ def extract_operation_io(
                     # Resolve response object safely with schema resolver
                     resolved_response = _resolve_schema_refs(success_response, spec)
```

Comment:
NOTE: we're using `_resolve_schema_refs` on OpenAPI spec objects different than Schema Object. Resolving Schema Object in OpenAPI 3.1 has different semantics apart from just resolving `Reference Object. I'm just noting it so that we know about it.